resources/views/pages-legacy/gameInfo.blade.php

```diff
@@ -110,6 +110,13 @@
 $isFullyFeaturedGame = System::isGameSystem($gameData['ConsoleID']);
 $isEventGame = $gameData['ConsoleID'] == System::Events;
 
+// future events can only be viewed by users who can manage events.
+if ($isEventGame && $gameModel->event?->active_from != null
+    && $gameModel->event->active_from > Carbon::now()
+    && !$userModel?->can('manage', $gameModel->event)) {
+    abort(401);
+}
```

Comment:
It would probably be best for the majority of this logic to live in `EventPolicy::view()`.

Reply:
Good idea. Done.